exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml

```diff
@@ -19,20 +19,25 @@
     state: present
     backrefs: yes
 
-- name: Enable SSH password authentication
+- name: Find SSH Config Files
+  shell: ls /etc/ssh/sshd_config.d/* /etc/ssh/sshd_config
+  register: ssh_config_files
```

Comment:
Why not using the builtin Ansible find task:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/find_module.html